ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -683,10 +709,17 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		let photoLibrary = UIAlertAction(title: "Upload from your photo library".localized, style: .default, handler: { (_) in
```

Comment:
Key is missing in localizable.strings

Reply:
@hosy didn't notice since code was already there.. Will fix.

Reply:
@hosy just searched for the key and found it, it was there.